server/handlers.go

```diff
@@ -142,7 +142,7 @@ func regParkingSpot(c *gin.Context) {
 		db.Save(&owner)
 	}
 
-	c.JSON(201, "Listed a new parking Spot Successfully!")
+	c.String(201, "Listed a new parking Spot Successfully!")
```

Comment:
See above comment

Reply:
Updated in the commit 46c5e524ee9090fa246d58ea5922440dd6f7147c